modules/sms_user/config/install/sms_user.settings.yml

```diff
@@ -1,6 +1,15 @@
 active_hours:
   status: false
   ranges: []
-registration_enabled: false
-allow_password: false
-new_account_message: ''
+account_registration:
+  all_unknown_numbers:
+    status: false
+    reply_message: ''
+  formatted:
+    status: false
+    incoming_messages: []
+    activation_email: true
+    reply:
+      status: false
+      message: ''
+      message_failure: ''
```

Comment:
Some ideas for naming:

**unrecognised** / unknown / new / unconfirmed / unbound / unassociated

**pattern match / pattern** / template (fill in the gaps)